lib/util.ex

```diff
@@ -75,24 +70,51 @@ defmodule Util do
 
     case HTTPoison.post("https://api.start.gg/gql/alpha", JSON.encode!(body), headers) do
       {:ok, resp} ->
-        resp.body
-        |> JSON.decode
-        |> format_decoded_response
+        tourney_map = resp.body
```

Comment:
Let's refrain from calling this variable (or the corresponding function) `..._map`. It's not really the map, just the data that the frontend could use to populate the map. More importantly, I'd like to avoid any readers confusing this for a map (as in dictionary) data structure.

Reply:
As an alternative, maybe `tourney_data`, `raw_json`, or similar? And the function would be `extract_data` or similar?

Also, since the `format_decoded_response` function is no longer being passed the complete response, maybe we should rename that to `format_raw_data` or something.